test/index-test.js

```diff
@@ -1,53 +1,40 @@
 /*global describe, it */
 
-const expect = require('chai').expect
-const fs = require('fs')
-const jsdom = require('mocha-jsdom')
-const path = require('path')
-
-const file = fs.readFileSync(path.resolve(__dirname, '..', 'index.js'), 'utf-8')
-
-describe('index', () => {
-  jsdom({
-    src: file
+describe('myAnimal()', () => {
+  it('returns my animal', () => {
+    expect(myAnimal()).to.equal('dog')
   })
+})
 
-  describe('myAnimal()', () => {
-    it('returns my animal', () => {
-      expect(myAnimal()).to.equal('dog')
-    })
+describe('yourAnimal()', () => {
+  it('returns your animal', () => {
+    expect(yourAnimal()).to.equal('cat')
```

Comment:
`.to.equal` will need to change to `toEqual` to work with `expect` (sorry, some of these early ones used different assertion libraries).